CONTRIBUTING.md

```diff
@@ -18,45 +18,163 @@ any additional information for contributing code to this project.
 
 ## Building
 
-### Egg format
+### Static/portable CLI binary
 
 ```
-$ ./bin/build
+$ ./bin/build_binary
 ```
 
-### Static/portable CLI binary
+### Egg format
 
 ```
-$ ./bin/build_binary
+$ ./bin/build
 ```
 
 ## Development
 
-- Create a directory that will hold all the virtualenv packages and files:
+To setup a development environment follow the instructions in this section. Once you have done so, you will be able to see live changes made to the CLI.
+
+1. Create a directory that will hold all the virtualenv packages and files. Note that you will only need to run this command once.
+
+macOS:
+
 ```
 $ python3 -m venv venv
 ```
 
-- Enable your terminal to use those files with this command:
+Windows:
+
+```
+py -m venv venv
+```
+
+2. Enable your terminal to use those files with this command. Note that this command will need to run each time you want to return to your virtual environment.
```

Comment:
i think it's best to use only `1` for lesser burden in adding future steps

Reply:
1 OS to show? These steps also help in our release process so I prefer to leave them because they are a good reference

Reply:
no. i meant using `1` for all steps and not use `2`